onebusaway-android/src/main/AndroidManifest.xml

```diff
@@ -37,6 +37,7 @@
     <uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT"/>
     <uses-permission android:name="${applicationId}.permission.TRIP_SERVICE"/>
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
+    <uses-permission android:name="com.google.android.gms.permission.ACTIVITY_RECOGNITION" />
```

Comment:
If I'm reading this correctly, if we target Android 9 (API Level 28) we'll need to request permissions at runtime for this, similar to location:
https://developer.android.com/preview/privacy/permissions

We're currently `targetSdkVersion 26` - we should think about whether we should implement runtime permissions now to prepare for this, or wait until later.

Reply:
Here's another consideration for targeting Android Q:

~~~
<uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
~~~

Looks like this will be necessary for us to get the location info after getting an activity transition callback when targeting Android Q:
https://developer.android.com/preview/privacy/device-location

However, we'd also need to request this as a runtime permission with `ACCESS_FINE_LOCATION` when enrolling a user in the study.

Reply:
With the latest set of changes, we do need to add `ACCESS_BACKGROUND_LOCATION` to the manifest if we're requesting it at runtime when enrolling in the study.

Reply:
Added `ACCESS_BACKGROUND_LOCATION` to the manifest.